README-dev.md

```diff
@@ -51,7 +51,7 @@ Refer to [/dev](/dev).
 Mina has a variety of opam and system dependencies.
 
 To get all the opam dependencies you need, you run `opam switch import
-src/opam.export`.
+opam.export`.
```

Comment:
btw this is on the wrong line (I tried copy/pasting this the other day and got some issue)